Api/OrderManagementInterface.php

```diff
@@ -104,9 +104,10 @@ public function createInvoice($id, $amount, $notify = false);
      * & we don't want to trigger some after events if transaction failed.
      *
      * @param int $id The order ID.
+     * @param array $additionalTransactionInfo Additional transaction information.
      * @return \Magento\Sales\Api\Data\OrderInterface
      * @throws NoSuchEntityException
      * @throws WebapiException
      */
-    public function placeOrder($id);
+    public function placeOrder($id, $additionalTransactionInfo);
```

Comment:
Method annotation/parameters is not correct:

This is called in this endpoint in webhook [here](https://github.com/BoltApp/source/blob/9236b53c0632b1e7e8d6d7fe9dd7897d1ec05fcc/hail/services/merchantproxy/cart/magento2/webhook.go#L177)

The method used to call this endpoint implemented [here](https://github.com/BoltApp/source/blob/9236b53c0632b1e7e8d6d7fe9dd7897d1ec05fcc/hail/bolt/magento2/client/live_client.go#L1351)

There in bolt implemetation we are sending additional parameter = `transaction_data`
- So input parameter in php method should be called = `$transactionData`
- Annotation type for this parameter in interface should be `mixed` because array type is not supported by magento.

So it should be :
`@param mixed $transactionData Additional transaction information.`

p.s. to debug this endpoint on local we need just place any order via m2 api driven approach, and also please be sure we have success results [here](https://github.com/BoltApp/source/blob/9236b53c0632b1e7e8d6d7fe9dd7897d1ec05fcc/hail/services/merchantproxy/cart/magento2/webhook.go#L171)
`result, currentVersion, compErr := CheckBoltVersion(ctx, db, &division, "2.26.4")`